packages/ui/src/components/PropertiesModal/PropertiesModal.tsx

```diff
@@ -50,6 +51,18 @@ export const PropertiesModal: FunctionComponent<IPropertiesModalProps> = (props)
     setActiveTab(tabs[tabIndex as number]);
     setActiveTabKey(tabIndex as number);
   };
+  const iconName = props.tile.nodeIconType === NodeIconType.Kamelet ? `kamelet:${props.tile.name}` : props.tile.name;
+
+  const title: ReactElement = (
+    <div style={{ display: 'flex' }}>
```

Comment:
There's a `PropertiesModal.scss` file, could we add this style in a CSS class there? :pray: